react-native-eslint.md

```diff
@@ -0,0 +1,121 @@
+# Introduction
+
+ESLint is a tool that allows us to maintain code quality and enforce code conventions. ESLint is a static code evaluator. Basically, it means that ESLint will not actually execute the code but will instead read through the source code to see if all the preconfigured code conventions are followed by the developers.
+
+At [Osmosys Software Solutions Pvt Ltd](https://osmosys.co/) we standardize the ESLint configuration, which will be used across multiple projects.
+
+# Prerequisite
+
+React Native app with JavaScript or TypeScript
+
+Any existing React Native app with JavaScript or TypeScript. If you don't have one, then you can create a basic React Native app using the following command.
+
+> **Note**: for now assuming we will use React Native app with JS
+
+```sh
+npm react-native init my_app
+```
+
+React Native comes with a predefined ESLint config, but sometimes it doesn’t come with the latest ESLint version, so we’ll simply uninstall it and then install it again to pull the latest version.
+
+```sh
+npm uninstall eslint
+
+# install the latest eslint package version
+npm install eslint --save-dev
+```
+
+# Installation
+
+Install the following packages:
+
+- [eslint-plugin-react@latest](https://github.com/jsx-eslint/eslint-plugin-react)
+- [@typescript-eslint/eslint-plugin@latest](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin)
+- [eslint-config-standard@latest](https://github.com/standard/eslint-config-standard)
+- [eslint-plugin-import@^2.25.2](https://github.com/import-js/eslint-plugin-import)
+- [eslint-plugin-n@^15.0.0 ](https://github.com/github/eslint-plugin-github)
+- [eslint-plugin-promise@^6.0.0 ](https://github.com/xjamundx/eslint-plugin-promise)
+- [@typescript-eslint/parser@latest](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/parser)
+- [eslint-plugin-react-native](https://github.com/intellicode/eslint-plugin-react-native)
+
+```sh
+npm i eslint-plugin-react@latest, @typescript-eslint/eslint-plugin@latest, eslint-config-standard@latest, eslint@^8.0.1, eslint-plugin-import@^2.25.2, eslint-plugin-n@^15.0.0, eslint-plugin-promise@^6.0.0, @typescript-eslint/parser@latest eslint-plugin-react-native
```

Comment:
Another comment, shouldn't this be installed as `devDependencies`?

Reply:
it'll be installed as devDependencies only

Reply:
You are running `npm i`, that will add to dependencies right, and not `devDependencies`?

Reply:
I don't see that change.